src/services/milkomedaInfo.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import config from "config";
+
+enum NetworkNames { // exemplary names here
```

Comment:
Better to re-use the NPM package that already defines names for the network

Reply:
I was thinking about it, but we don't have it published yet - will that happen in near future? It didn't seem natural to add another project here and reference locally (when we know we'll have it accessible from npm soon).